searchengine/app.py

```diff
@@ -27,6 +27,12 @@ def get_symbols(id):
     return syms
 
 
+@lru_cache(maxsize=2000)
```

Comment:
Following `get_symbols()` I've added `@lru_cache` here as well. If the cache there is helpful then this cache here should be as well.